src/libcharon/encoding/payloads/eap_payload.c

```diff
@@ -245,11 +246,11 @@ static size_t extract_type(private_eap_payload_t *this, size_t offset,
 }
 
 METHOD(eap_payload_t, get_type, eap_type_t,
-	private_eap_payload_t *this, uint32_t *vendor)
```

Comment:
Can you separate out all the `uint32_t` to `eap_vendor_t` rewrites you did in its own commit please?

Reply:
Ok